ffc-pay-event/signalr.js

```diff
@@ -0,0 +1,11 @@
+const sendMessage = (context, message) => {
```

Comment:
could we change the name of this from `sendMessage` just to make it clearer we're talking about SignalR.  Maybe `sendMessageToSignalR` or something to make it clear that it's not just a typical send to service bus we have elsewhere in our services.